pkgs/desktops/xfce/default.nix

```diff
@@ -58,17 +58,13 @@ xfce_self = rec { # the lines are very long but it seems better than the even-od
   parole          = callPackage ./applications/parole.nix { };
   ristretto       = callPackage ./applications/ristretto.nix { };
   terminal        = xfce4terminal; # it has changed its name
-  xfce4mixer      = callPackage ./applications/xfce4-mixer.nix {
-    pulseaudioSupport = config.pulseaudio or false;
-  };
+  xfce4mixer      = callPackage ./applications/xfce4-mixer.nix { };
   xfce4notifyd    = callPackage ./applications/xfce4-notifyd.nix { };
   xfce4taskmanager= callPackage ./applications/xfce4-taskmanager.nix { };
   xfce4terminal   = callPackage ./applications/terminal.nix { };
   xfce4-screenshooter = callPackage ./applications/xfce4-screenshooter.nix { };
-  xfce4volumed    = let
```

Comment:
I think we should keep `xfce4volumed` for people that aren't using NixOS and are still using xfce applications from nixpkgs. Ditto for keeping the default for `pulseaudioSupport` for `xfce4mixer` above. With the changes in the nixos xfce module you made, this should have no effect on nixos but leave everything as it was before on non-nixos.